src/app/Fake.Core.String/String.fs

```diff
@@ -115,6 +115,13 @@ let inline appendIfNotNullOrEmpty value s = appendIfTrue (isNotNullOrEmpty value
 let inline appendFileNamesIfNotNull fileNames (builder : StringBuilder) = 
     fileNames |> Seq.fold (fun builder file -> appendIfTrue (isNullOrEmpty file |> not) file builder) builder
 
+/// Applies action on builder for each element of list.
+let inline forEach items action text (builder: StringBuilder) =
+    items
+    |> List.iter (fun t -> builder |> (action t text) |> ignore)
+
+    builder
```

Comment:
Yes, ```StringBulder```-related functions could be moved out of ```Fake.Core.String``` into separate module. 

Another question here - should ```Fake.StringHelper``` be marked as "obsolete" - because code of it duplicates ```Fake.Core.String```?